ui/src/main/resources/edu/wpi/grip/ui/MainWindow.fxml

```diff
@@ -96,6 +96,23 @@
                     </accelerator>
                 </MenuItem>
             </Menu>
+            <Menu text="Tools">
+                <MenuItem text="Settings" onAction="#showProjectSettingsEditor">
+                    <graphic>
+                        <ImageView styleClass="menu-graphic">
+                            <fitWidth>
+                                <DPIUtility fx:constant="SMALL_ICON_SIZE"/>
+                            </fitWidth>
+                            <fitHeight>
+                                <DPIUtility fx:constant="SMALL_ICON_SIZE"/>
+                            </fitHeight>
+                            <image>
+                                <Image url="@icons/settings.png"/>
```

Comment:
Okay, I messed with it enough and finally figured it out:
```
<Glyph fontFamily="FontAwesome" icon="UPLOAD" style="-fx-text-fill: black;"/>
```
The key is the style otherwise it will be transparent (at least in the menus)